helm-charts/arkmq-org-activemq-artemis-operator/values.yaml

```
@@ -0,0 +1,150 @@
+        digest: sha256:2d155bffd246f26f516a6075ec170babc1373c72bb664cba4e3c58521ffd3558
+      activemqArtemisBrokerKubernetes2260:
+        digest: sha256:f25db980c3229e66cf48690aa175f6f93e3dce3974c0e70ab23af6462e7d1893
+      activemqArtemisBrokerKubernetes2270:
+        digest: sha256:a4628c2591ec06caf21a3467f6e3faeaede7fa402b4f137d943b25b4dd4ad343
+      activemqArtemisBrokerKubernetes2271:
+        digest: sha256:2895b2cf6292c951a0cdde78e755514f5d7cb43ce9f7b6e7dc381d8dfeec31ab
+      activemqArtemisBrokerKubernetes2280:
+        digest: sha256:42230ce9d11a1949a5bb73b6ee218b1581f585bcd553485ee438e1c5b50fcf8b
+      activemqArtemisBrokerKubernetes2290:
+        digest: sha256:4be60cfa2a300b2eef9b5f10ea756bd3f55d75fbd8890232d907f8d64080449b
+      activemqArtemisBrokerKubernetes2300:
+        digest: sha256:8fbdd1ba728058d5f437186b9055ca28596140068ebc69ecff4fa52717b19a82
+      activemqArtemisBrokerKubernetes2310:
+        digest: sha256:4bba8828e72fd7c9d38c22ba1c2a4f7130817a4e7eac8fb63c13e86465a8e1e7
+      activemqArtemisBrokerKubernetes2312:
+        digest: sha256:81590ec33b2f00775d3c6596f610887b670de7c0b89148b09eab7584f5b0b0ec
+      activemqArtemisBrokerKubernetes2320:
+        digest: sha256:52f337cd5d7d5d199eebf4fc722f8c5279f652506f009efbc0f8f6de88e18718
+      activemqArtemisBrokerKubernetes2330:
+        digest: sha256:b293c1269166cc6181f68b8d803fdfcf20fdbd5df2787ed0453724b22e298fb2
+      activemqArtemisBrokerKubernetes2340:
+        digest: sha256:6f2da9ecd8f5021964d0dfa85144830efcfbd0462d18f18f036971e5f17b3bb6
+      activemqArtemisBrokerKubernetes2350:
+        digest: sha256:49f34122ab46ab5d7ead094b6e40817b62c75b6a809f67db07d68bce750da51e
+      activemqArtemisBrokerKubernetes2360:
+        digest: sha256:9e1fefc9a94d72a7b5a3f4e2d53ebd23e5ab221df17a5afde42ce973277838f6
+      activemqArtemisBrokerKubernetes2370:
+        digest: sha256:5e768aefade6d18cd65cb1eb5ee233844f9f7e7b52631f93a0982b67d382239d
+      activemqArtemisBrokerKubernetes2380:
+        digest: sha256:37ab7354273f4dcd19bad6936f10e92220d98b8c0266002221477a0e2d146c81
+      activemqArtemisBrokerKubernetes2390:
+        digest: sha256:70d4c799c7d905fac27457d63e662ea72bca9e899a692e425609e4eaa45129e6
+      activemqArtemisBrokerKubernetes2400:
+        digest: sha256:a4a0d0453f0761b9e114d0d4cc7c14091a2a9c30790f18ffa6bf1e365d5083fe
+      activemqArtemisBrokerKubernetes2410:
+        digest: sha256:decb3932ebb5b848b90a4a23b083f0cb5237eec2844095d8c58a6798da8bb999
+      activemqArtemisBrokerKubernetes2420:
+        digest: sha256:673384959d87408cab9fc92f305dc47b3e7bad3a81b182de104d47f76e7df4e1
+    resources: null
+  podSecurityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  replicas: 1
+  serviceAccount:
+    annotations: {}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+imagePullSecrets: []
+kubernetesClusterDomain: cluster.local
+clusterWide: false
+watchNamespace: null
```

We could replace `clusterWide` and `watchNamespace` with `watchNamespaces`:
- `watchNamespaces: null`, the operator watches all namespaces in the cluster;
- `watchNamespaces: []`, the operator watches only the release namespace;
- `watchNamespaces: ['foo']`, the operator watches only the namespaces in the array.

The default should be to watch all namespaces in the cluster:
```suggestion
watchNamespaces: null
```

---

@brusdev Already done!